README.md

```diff
@@ -1,34 +1,70 @@
 <h1 align="center"><a href="https://kubesimplify.github.io/website/">Kubesimplify Website </a></h1>
 <p align="center"><i>The source code for Kubesimplify.com</i></p>
+Kubesimplify's official website. This codebase is what runs on [Kubesimplify](https://kubesimplify.github.io/website/). 
 
-This repository contains the assets required to build the Kubesimplify website and documentation. We're glad that you want to contribute! 
+This repository contains the assets required to build the Kubesimplify website and documentation.
 
+## Contribute
+We're glad that you want to contribute! 
+A first step to contributing is to start with something you think you can add value in te Community. Start attending Kubesimplify meetings, join the [Discord channels](https://discord.gg/uFBC8MyeeU) and subscribe to the [official youtube Channel](https://www.youtube.com/c/saiyam911). Ask in community, We always Welcome new Contributors.
```

Comment:
```suggestion
A first step to contributing is to start with something you think you can add value in the community. Start attending Kubesimplify meetings, join the [Discord channels](https://discord.gg/uFBC8MyeeU) and subscribe to the [official youtube Channel](https://www.youtube.com/c/saiyam911). Ask in community, We always Welcome new Contributors.
```

Reply:
@pavangudiwada are we suppose to add this para in the readme as of now or maybe can you rephrase this to something that we should say to community folks who are not the ambassadors